code/_globalvars/misc.dm

```diff
@@ -62,4 +62,6 @@ var/score_richestkey = null
 var/score_dmgestname = null // who had the most damage on the shuttle (but was still alive)
 var/score_dmgestjob = null
 var/score_dmgestdamage = 0
-var/score_dmgestkey = null
\ No newline at end of file
+var/score_dmgestkey = null
+
+var/TAB = "&nbsp;&nbsp;&nbsp;&nbsp;"
```

Comment:
... will this even work? Is byond/internet explorer really shit enough to not suppress white-spaces?